src/lib/Load.mli

```diff
@@ -1,4 +1,4 @@
-type error = LexingError of Basis.LexingUtil.span | ParseError of Basis.LexingUtil.span
+type error = LexingError of Basis.LexingUtil.span * string | ParseError of Basis.LexingUtil.span * string
```

Comment:
I suggest changing this to be an inline record so that the intention of the `string` thing is clear from the interface. We could write something like:

```ocaml
type error = 
| LexingError of {span : ...; last_token : string}
| ...
```

Reply:
I cleared this up in the latest version.